Setup/Patch/Data/BlockFulfillmentStatusCreate.php

```diff
@@ -7,7 +7,7 @@
 
 class BlockFulfillmentStatusCreate extends SimpleDataPatch
 {
-    public function apply(): self
+    public function apply(): DataPatchInterface
     {
         $this->block->save([
```

Comment:
I think we should `return $this` within this apply function, no?

Reply:
@markshust good catch! Thanks.